src/Firehose.Web/Authors/GiampaoloTucci.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class GiampaoloTucci : IAmACommunityMember, IAmAPodcast
+    {
+        public Uri WebSite => new Uri("https:/www.informaticapressapochista.com/it");
+        public string FirstName => "Giampaolo";
+        public string LastName => "Tucci";
+        public string StateOrRegion => "Italia";
+        public string EmailAddress => "g.tucci@informaticapressapochista.com";
+        public string ShortBioOrTagLine => "L'IT come non l'avete mai letta";
+        public string GravatarHash => "26850a95739983a8280262b0044fe3e5";
+        
+        public IEnumerable<Uri> FeedUris
+        {
+            get { yield return new Uri("https://www.informaticapressapochista.com/it/?format=feed&type=rss"); }
```

Comment:
Your feed doesn't seem to be parsable by BlogMonster, which we use to parse feeds. Sorry, but I don't think we are going to change this right now.

Reply:
Nevermind, it is actually the filtering that filters away all your posts. Right now it returns an empty feed, which we don't accept.